src/Controls/tests/UITests/Tests/Issues/Issue19127.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue19127 : _IssuesUITest
+	{
+		public override string Issue => "[iOS] Triggers are not working on Frame control";
+
+		public Issue19127(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		public void ContentOfFrameShouldChange()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Mac, TestDevice.Windows });
```

Comment:
I know this is an iOS issue, but I think this test could have sense to validate the same behavior on other platforms. Could we pass the test also on Android and Windows?

Reply:
Of course